packages/app-desktop/gui/NoteEditor/NoteEditor.tsx

```diff
@@ -632,9 +644,30 @@ function NoteEditorContent(props: NoteEditorProps) {
 
 	const theme = themeStyle(props.themeId);
 
+	function renderConvertHtmlToMarkdown(): React.ReactNode {
+		if (!props.showConvertHtmlToMarkdownMessage) return null;
+
+		const note = props.notes.find(n => n.id === props.selectedNoteIds[0]);
+		if (!note) return null;
+		if (note.markup_language !== MarkupLanguage.Html) return null;
+
+		return (
+			<div style={styles.resourceWatchBanner}>
+				<p style={styles.resourceWatchBannerLine}>
+					{_('This note is in HTML format and may be be difficult to edit. Convert it to Markdown to edit it more easily.')}
```

Comment:
This note is in HTML format. Convert it to Markdown to edit it more easily.
```suggestion
					{_('This note is in HTML format. Convert it to Markdown to edit it more easily.')}
```